deploy/deploy.go

```diff
@@ -233,7 +284,36 @@ func (k *KindSpec) checkDependencies() error {
 			errs.Add(fmt.Errorf("install dependency %q to deploy", bin))
 		}
 	}
-	return errs.Err()
+	if errs.Err() != nil {
+		return errs.Err()
+	}
+	if k.Version != "" {
```

Comment:
Where does Version even come into play? Currently its a field in the yaml but I can't find where it is used. 

If the user does not provide a version what should happen?

Reply:
it is used right below?

Reply:
if none set - do nothing assume the user knows what they are doing